README.md

```diff
@@ -1,3 +1,19 @@
+<!-- 
+  Hivemall: Hive scalable Machine Learning Library
```

Comment:
Yes, it should have a common license header.
https://github.com/myui/hivemall/blob/jira/HIVEMALL-12/resources/license-header.txt